script/main.js

```diff
@@ -0,0 +1,157 @@
+"use strict";
+
+class StockSearch {
```

Comment:
in the project instractions you were asked to seperate the logic of the searchForm into a class and the logic of searchResults into another one. Here he have a combination of both... Please see the ITC solution when published in order to understand better how your code should look like.